### ISSUE TITLE
Language server stops working when a non-ASCII character is used in a command/macro name

### ISSUE THREAD

**Issue by Andrew15-5** [latex-lsp/texlab#857]
This is a copy of https://github.com/fannheyward/coc-texlab/issues/249 because the problem is not with coc-texlab but with texlab itself.

```latex
\documentclass{article}
% \newcommand\ö{}
% \def\ö{}
\def\123{}
\def\abc{}
\begin{document}
\end{document}
```

If comments are present, then the autocompletion works in Neovim, but when any of the comments are uncommented then coc-texlab (texlab) stops working. If I redo the comments, then it will show autocompletions again.

![works](https://user-images.githubusercontent.com/37143421/221281666-04509f7b-cffe-410b-9717-39e43650c2fa.png)
![doesn't work](https://user-images.githubusercontent.com/37143421/221281709-6797ff81-8b49-4657-bc2e-8c0a320050b7.png)
![doesn't work](https://user-images.githubusercontent.com/37143421/221281754-71d63a58-45ca-48a2-8c29-a246366f06c9.png)

Using only:
```vim
" init.vim
call plug#begin($HOME . '/.config/nvim/autoload/plugged')
  Plug 'neoclide/coc.nvim', {'branch': 'release'}
call plug#end()
```

`\nvim -u ./init.vim`:

![image](https://user-images.githubusercontent.com/37143421/221282583-954eba87-79b9-4a64-94ae-7b04318bd93b.png)
![image](https://user-images.githubusercontent.com/37143421/221282632-ae9aef24-a556-45f4-8b73-68abe4dba3c4.png)
![image](https://user-images.githubusercontent.com/37143421/221282696-21547b4f-36e4-4603-a4d2-09531ced4e90.png)

**Comment by pfoerster** [comment]
@Andrew15-5 Thanks for the report. This should be fixed now with 51f7179.

**Comment by Andrew15-5** [comment]
Wow, thank you so much!
